src/score_gen/score.rs

```diff
@@ -446,22 +446,7 @@ fn bits_count(mut scalar: BlsScalar) -> u64 {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use dusk_plonk::jubjub::{
-        AffinePoint, GENERATOR_EXTENDED, GENERATOR_NUMS_EXTENDED,
-    };
-    use rand_core::RngCore;
-
-    pub(self) fn gen_val_blinder_and_commitment(
-    ) -> (JubJubScalar, JubJubScalar, AffinePoint) {
-        let value = JubJubScalar::from(250_000u64);
-        let blinder = JubJubScalar::random(&mut rand::thread_rng());
-
-        let commitment: AffinePoint = AffinePoint::from(
-            &(GENERATOR_EXTENDED * value)
-                + &(GENERATOR_NUMS_EXTENDED * blinder),
-        );
-        (value, blinder, commitment)
-    }
```

Comment:
Same as avobe. Not sure why we remove this.

Reply:
Check above

Reply:
LGTM